### ISSUE TITLE
Contributing Guidelines?

### ISSUE THREAD

**Issue by dlaird-ovo** [K-Phoen/backstage-plugin-confluence#175]
I am thinking about adding some new functionality to the plugin.
It's currently in the hack/explore phase.

However I am struggling to get my own Backstage App to link to a local checkout of this package and allow easy development.

I wondered how your flow works, and how you do your build/dev cycle.

**Comment by seanblonien** [comment]
Dynamic spaces is such a good idea!! we definitely have that use case (I feel like everyone has the use case for spaces being dynamic...) and would love to see that feature

I am also interested in contributing a bit as well as I am making another custom backstage plugin and believe I could help develop this one as well

@K-Phoen do you have any intention about making this more open to contributors with some defined contributor guidelines?

**Comment by K-Phoen** [comment]
@seanblonien I'm working on this plugin purely as a kind of "hobby project", pretty much whenever I feel like it and on what issue/feature interests me at the time. As such, and I agree with you, documentation & DX is lacking and it's not the most contributor-friendly repository on GitHub.

If you do use this plugin in your company and would want to see some work reliably prioritized, may I suggest sponsoring some of that work?

**Comment by seanblonien** [comment]
@K-Phoen good idea. I will propose that as an option, especially if we are wanting extended features from this plugin. otherwise we may just be submitting PRs which would still be helpful
